.travis.yml

```diff
@@ -210,7 +219,7 @@ install:
   # - npm install
   # but since there are often connection failures, use the retry package.
   - npm-install-retry
-  - BABEL_ENV=cover NYC_CWD=$large_image_path girder-install web --plugins=jobs,worker,large_image --dev
```

Comment:
Is the `BABEL_ENV` variable not necessary any more?

Reply:
https://github.com/girder/girder/commit/93064d13c10440e41722c3d2e4330c9306e91772

Reply:
Correct.  Girder PR https://github.com/girder/girder/pull/2500 had it no longer necessary and moved where coverage files are emitted.